### PR TITLE
Split Ubuntu build images

### PR DESCRIPTION
**Pull request by KVS85**

### CODE REVIEW

File: containers/buildUbuntu2.docker/scripts/dummy [hunk 1]
```diff
@@ -0,0 +1 @@
+A file to keep the directory.
```
Comment (Simran-B) [inline containers/buildUbuntu2.docker/scripts/dummy:1]: If this is for Git, then the usual convention is to call the file `.gitkeep`

**Comment by KVS85** [comment]
3.4: http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/9858/
3.5: http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/9859/
3.6: http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/9860/
devel (3.7): 

3.6 ASAN: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-asan/260/

File: containers/buildUbuntu2.docker/scripts/dummy (hunk 1 above)
Comment (joerg84) [reply containers/buildUbuntu2.docker/scripts/dummy:1]: +1

File: containers/buildUbuntu.docker/tools/install.sh [hunk 2]
```diff
@@ -14,16 +15,28 @@ update-alternatives --set cc /usr/bin/gcc
...
+echo $OPENSSLBRANCH
+
+if [ "$OPENSSLBRANCH" != "1.1.1" ]; then
+  OLD="old/${OPENSSLBRANCH}/"
+fi;
+
+echo "https://www.openssl.org/source/${OLD}openssl-$OPENSSLVERSION.tar.gz"
+
+test -n "$OPENSSLVERSION"
+export OPENSSLPATH=`echo $OPENSSLVERSION | tr -d "a-zA-Z"`
```
[the remark this answers is not in the document]
Comment (KVS85) [reply containers/buildUbuntu.docker/tools/install.sh:32]: Yes, just a syntactic variable to not mess path of target build.

File: containers/buildUbuntu.docker/tools/install.sh [hunk 3]
```diff
@@ -14,16 +15,28 @@ update-alternatives --set cc /usr/bin/gcc
...
+export OPENSSLREVISION=$2
+export OPENSSLVERSION=${OPENSSLBRANCH}${OPENSSLREVISION}
+
+echo $OPENSSLBRANCH
+
+if [ "$OPENSSLBRANCH" != "1.1.1" ]; then
+  OLD="old/${OPENSSLBRANCH}/"
+fi;
+
+echo "https://www.openssl.org/source/${OLD}openssl-$OPENSSLVERSION.tar.gz"
```
[the remark this answers is not in the document]
Comment (KVS85) [reply containers/buildUbuntu.docker/tools/install.sh:29]: Leftover. Will be removed.

File: containers/buildUbuntu.docker/tools/install.sh [hunk 4]
```diff
@@ -14,16 +15,28 @@ update-alternatives --set cc /usr/bin/gcc
...
+
+test -n "$OPENSSLVERSION"
```
[the remark this answers is not in the document]
Comment (KVS85) [reply containers/buildUbuntu.docker/tools/install.sh:31]: Yes. That is just leftover. Will be removed.

**Comment by KVS85** [comment]
devel ASAN: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-asan/266/
3.6 ASAN: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-asan/267/
3.5 ASAN: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-asan/268/
3.4 ASAN: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-asan/269/

**Comment by KVS85** [comment]
devel: http://jenkins.arangodb.biz:8080/job/arangodb-ANY-linux-asan/270/

**Pull request merged**